src/components/token-symbol.js

```diff
@@ -1,10 +1,6 @@
-import React from "react";
-import t from "prop-types";
-import { drizzleReactHooks } from "@drizzle/react-plugin";
+import t, { PropTypes } from "prop-types";
 
-const { useDrizzleState } = drizzleReactHooks;
-
-export default function TokenSymbol({ chainId, token }) {
+export function getTokenSymbol(chainId, token) {
```

Comment:
Considering where this function is being used, do you think it would be worth to make it a hook (using `useMemo` and getting the `chainId` directly)? 🤔 There are a couple of files where the function is used in the return statement, but I think it could be moved upwards like in `pnk-actions` 🤔

Reply:
that's a good idea but, sometimes you want to get the token symbols of other chains, like here:

![image](https://github.com/kleros/court/assets/102478601/233870b0-1ea7-4184-878f-b209ab032fa8)

I feel like I might stick to adding useMemos? what do you think

Reply:
ooh, yep makes sense 👍🏻